docs/code/Flakes/Reference/Schema.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Nix Flake Schema
+date: 2021-02-09
+---
+
+The `flake.nix` has the following schema:
```

Comment:
I have a feeling this will go out of date, and the original sources could just be linked.
On the other hand, the flake schema is not well documented.
The resources I use are:
1. Schema - https://nixos.wiki/wiki/Flakes#Flake_schema
2. Input references - https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake.html#flake-references
3. Installables - https://nixos.org/manual/nix/stable/command-ref/new-cli/nix.html#installables